calico/src/main/scala/calico/html/NodesInterpolator.scala

```diff
@@ -122,20 +122,17 @@ given nodesInterpolatorModifier[E <: HtmlElement[IO]]: Modifier[IO, E, NodesInte
                   span
                 })
                 .flatMap { span =>
-                  Resource
-                    .make(
-                      signal
-                        .discrete
-                        .foreach { value =>
-                          IO {
-                            span.textContent = value.toString
-                          }
-                        }
-                        .compile
-                        .drain
-                        .start
-                    )(fiber => fiber.cancel)
-                    .void
+                  val stream = signal
+                    .discrete
+                    .foreach { value =>
+                      IO {
+                        span.textContent = value.toString
+                      }
+                    }
+                    .compile
+                    .drain
+
+                  stream.background.as(())
```

Comment:
`as(())` is the definition of `void` :)

```suggestion
                  stream.background.void
```

Reply:
apologize for this much changes :( understood so many things thankyou, willing to contribute more :)

Reply:
if needs any more changes please tell :)